srcJava/com/rti/perftest/ddsimpl/RTIDDSImpl.java

```diff
@@ -952,11 +937,10 @@ private void configureWriterQos(
                     _sendQueueSize;
         }
 
-        if (("LatencyQos".equals(qosProfile) ||
-             "NoAckLatencyQos".equals(qosProfile)) &&
-             !_directCommunication &&
-             (_durability == 2 ||
-              _durability == 3)){
+        if ("LatencyQos".equals(qosProfile)
+                && !_directCommunication 
+                && (_durability == 2
+                    || _durability == 3)){
```

Comment:
why dont we use:
- TRANSIENT_DURABILITY_QOS instead of 2
- PERSISTENT_DURABILITY_QOS instead of 3

REF: https://community.rti.com/static/documentation/connext-dds/5.3.0/doc/api/connext_dds/api_java/classcom_1_1rti_1_1dds_1_1infrastructure_1_1DurabilityQosPolicyKind.html#a0b69538f57900f159c71dce3b47890c0

Reply:
Have you fixed this one?